tpchgen/src/text.rs

```diff
@@ -62,22 +62,14 @@ impl TextPool {
         self.size
     }
 
-    /// Returns a chunk of text from the pool.
-    pub fn text(&self, begin: i32, end: i32) -> String {
-        assert!(begin >= 0, "Begin index must be greater than or equal to 0");
```

Comment:
I removed these asserts as they are redundant in Rust -- by default `&data[begin..end]` will check that `begin..end` are within bounds and `panic` if they are not.